zerver/openapi/zulip.yaml

```diff
@@ -5450,6 +5450,86 @@ paths:
                         "result": "success",
                         "msg": "",
                       }
+  /users/{email}:
+    get:
+      operationId: get_user_by_email
+      tags: ["users"]
+      description: |
+        Fetch details for a single user in the organization.
+
+        `GET {{ api_url }}/v1/users/{email}`
+
+        You can also fetch details on [all users in the organization](/api/get-users).
+
+        *This endpoint is new in Zulip Server 3.0 (feature level 1).*
+      parameters:
+        - name: email
+          in: path
+          description: |
+            The email address of the user whose presence you want to fetch.
```

Comment:
This description is copy-pasted and suggests we're fetching presence, which isn't true in this request.